mysql-test/suite/rocksdb/include/type_varchar_endspace.inc

```diff
@@ -42,18 +42,24 @@ insert into t1 values (0, 'ab', 'a-b');
 insert into t1 values (1, 'a ', 'a-space');
 insert into t1 values (2, 'a',  'a');
 insert into t1 values (3, 'a  \t', 'a-tab');
+# force cardinality updates otherwise explains below are non-deterministic
+# and will change query plans with so few rows.
+analyze table t1;
 
 --echo # Must show 'using index' for latin1_bin and utf8_bin:
 --replace_column 10 # 11 #
 explain
-select col1, hex(col1) from t1;
-select col1, hex(col1) from t1;
+select col1, hex(col1) from t1 order by col1;
+select col1, hex(col1) from t1 order by col1;
 
 --echo # Must show 'using index' for latin1_bin and utf8_bin:
 --replace_column 5 # 7 # 8 # 10 # 11 #
 explain
-select col1, hex(col1) from t1 where col1 < 'b';
-select col1, hex(col1) from t1 where col1 < 'b';
+select col1, hex(col1) from t1 where col1 < 'b' order by col1;
+#set optimizer_trace="enabled=on";
+select col1, hex(col1) from t1 where col1 < 'b' order by col1;
+#select * from information_schema.optimizer_trace;
```

Comment:
Ohh crap, that was some of my debug code trying to figure out where the instability was coming from. Should have never made it into the push.